tests/jest/unit/app/modules/import-export/assets/js/admin.test.js

```diff
@@ -0,0 +1,121 @@
+describe( 'Import Export Admin Test', () => {
+	window.elementorAppConfig = {
+		'import-export': {
+			lastImportedSession: {
+				kit_title: 'Kit Title',
+			},
+		},
+	};
+
+	require( 'elementor/app/modules/import-export/assets/js/admin.js' );
+
+	it( 'should check if "kit deleted" dialog is not shown without cache', () => {
+		// Arrange
+		window.elementorCommon = {
+			dialogsManager: {
+				createWidget: jest.fn( () => {
+					return {
+						show: jest.fn( () => {} ),
+					};
+				} ),
+			},
+		};
+
+		// Act
+		window.dispatchEvent( new Event( 'load' ) );
```

Comment:
I think it'll be easier to unit test the Admin class only, so you can just:
```JS
// Act
( new Admin() ).init()
```
or something

Reply:
I am not exporting the class, and if the whole class is meant to run on load like in our case, essentially the same thing as doing `new Admin()` I don't see what is wrong with this